src/adera_drawing_gl/flat_shader.h

```diff
@@ -24,7 +24,7 @@
  */
 #pragma once
 
-#include <osp/drawing_gl/rendergl.h>
+#include <osp_drawing_gl/rendergl.h>
```

Comment:
i guess i'm not following why you're changing the folder hierarchy like this?

is having a subdirectory causing a problem?

Reply:
Mostly, I don't want to compile the drawing_gl part when building the GdExtension, to not depend on Magnum GL, SDL, and the like. 

I figured it would be clearer to separate the common code, magnum-standalone-specific code, and Godot code in different directories instead of hiding it down the folder hierarchy. Also saves quite a few CMakeLists.txt files.